File: remove_bg.py
from PIL import Image

def remove_bg(img_path):
    img = Image.open(img_path).convert("RGBA")
    datas = img.getdata()
    newData = []
    for item in datas:
        # If the pixel is very dark, make it transparent
        if item[0] < 15 and item[1] < 15 and item[2] < 15:
            newData.append((0, 0, 0, 0))
        else:
            newData.append(item)
    img.putdata(newData)
    img.save(img_path, "PNG")
    print("Background removed successfully.")

remove_bg("public/exismic-app-icon.png")

File: remove_bg.js
const Jimp = require('jimp');

Jimp.read('public/exismic-app-icon.png')
  .then(img => {
    img.scan(0, 0, img.bitmap.width, img.bitmap.height, function (x, y, idx) {
      const red   = this.bitmap.data[idx + 0];
      const green = this.bitmap.data[idx + 1];
      const blue  = this.bitmap.data[idx + 2];
      
      // If color is close to black
      if (red < 20 && green < 20 && blue < 20) {
        this.bitmap.data[idx + 3] = 0; // Set alpha to 0 (transparent)
      }
    });
    return img.writeAsync('public/exismic-app-icon.png');
  })
  .then(() => {
    console.log("Background removed successfully.");
  })
  .catch(err => {
    console.error("Error processing image:", err);
  });
